applications/StructuralMechanicsApplication/tests/InitialStateElasticity/initial_strain_cr_beam3D2N_test_parameters.json

```diff
@@ -0,0 +1,164 @@
+{
+    "problem_data"     : {
+        "problem_name"  : "initial_strain_cr_beam3D2N_test",
+        "parallel_type" : "OpenMP",
+        "echo_level"    : 1,
+        "start_time"    : 0.0,
+        "end_time"      : 1.0
+    },
+    "solver_settings"  : {
+        "solver_type"                     : "Static",
+        "model_part_name"                 : "Structure",
+        "domain_size"                     : 3,
+        "echo_level"                      : 1,
+        "analysis_type"                   : "non_linear",
+        "model_import_settings"           : {
+            "input_type"     : "mdpa",
+            "input_filename" : "initial_strain_cr_beam3D2N_test"
+        },
+        "material_import_settings"        : {
+            "materials_filename" : "initial_strain_cr_beam3D2N_test_materials.json"
+        },
+        "time_stepping"                   : {
+            "time_step" : 0.1
+        },
+        "line_search"                     : false,
+        "convergence_criterion"           : "displacement_criterion",
```

Comment:
Did you check this with the `and_criterion`?

Reply:
Yes. It works, with proper tolerances. I have just pushed a modified test with the `and_criterion`